tests/max_config/nidaqmxMaxConfig.ini

```diff
@@ -105,6 +105,22 @@ BusType = PCIe
 PCI.BusNum = 0x0
 PCI.DevNum = 0x0
 
+[DAQmxDevice nidaqmxUSBTester]
+ProductType = USB-9265
+DevSerialNum = 0x0
+DevIsSimulated = 1
+ProductNum = 0x712C
+BusType = USB
+
+[DAQmxDevice aoTester]
+ProductType = PXIe-4322
+DevSerialNum = 0x0
+DevIsSimulated = 1
+ProductNum = 0x76ABC4C4
+BusType = PXIe
+PXI.ChassisNum = 1
+PXI.SlotNum = 2
```

Comment:
bridgeTester is in chassis 1 slot 2, so move this to slot 3. (I think configuring multiple PXI devices in the same chassis/slot may produce routing errors in some cases.)

Reply:
Updated `SlotNum` of `aoTester` to 3.